docs/TESTING.md

```diff
@@ -17,7 +17,7 @@ GRADESCOPE_CI_TA_PASSWORD
 
 For test cases to pass:
 
-Student accounts are expected to be accounts that are **only** enrolled as students in courses. Similarly, instructor accounts are expected to **only** be instructors for courses. TA accounts are expected to be **both**. 
+Student accounts are expected to be accounts that are **only** enrolled as students in courses. Similarly, instructor accounts are expected to **only** be instructors for courses. TA accounts are expected to be **both**. Tests can also be skipped by using the pytest decorator ```@pytest.mark.skip(reason="...")```
```

Comment:
Maybe we can add the skipif decorator to skip tests that require valid credentials if env variables are not set.

Reply:
I think we can skip implementing this for now unless someone wants to do it